apps/hash-ai-worker-py/poetry.lock

```diff
@@ -418,9 +418,7 @@ version = "0.0.0"
 description = "Type definitions for the type system and entity validation based on schemas"
 optional = false
 python-versions = ">=3.11,<4.0"
-files = [
-    {file = "hash_graph_types-0.0.0-py3-none-any.whl", hash = "sha256:c03c391cf843e36bc392c812fb9924286377c523ccd294b6d3eeff08119c2e42"},
-]
+files = []
```

Comment:
Why is that here? I assume you forgot to lock again?

Reply:
Because this way it's working without messing around with the lock file. 🤷🏻‍♂️ 
I will most likely remove the wheel, anyway as it's super annoying to deal with.